packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -284,13 +284,15 @@ class CodePushClient {
     required String appId,
     required String version,
     required String flutterRevision,
+    String? flutterVersion,
```

Comment:
Since we've modified the code_push_client we should also bump the version in the pubspec.yaml and re-run build_runner to adjust the version.dart